pulp_container/tests/functional/api/test_push_content.py

```diff
@@ -169,6 +173,9 @@ def test_push_with_no_perms(
     """
     Test that user with no permissions can't perform push.
```

Comment:
maybe we can still run it even with token_auth disabled, with user_helpless

Reply:
i.e check with user helpless that he cannot tag_and push, check with user helpless that he can still pull repo whether it's private or public